src/main/java/uk/gov/hmcts/darts/transcriptions/mapper/TranscriptionResponseMapper.java

```diff
@@ -40,4 +45,31 @@ TranscriptionUrgencyResponse mapToTranscriptionUrgencyResponse(TranscriptionUrge
         transcriptionUrgencyResponse.setDescription(transcriptionUrgencyEntity.getDescription());
         return transcriptionUrgencyResponse;
     }
+
+    public static TranscriptionResponse mapToTranscriptionResponse(TranscriptionEntity transcriptionEntity) {
+
+        TranscriptionResponse transcriptionResponse = new TranscriptionResponse();
+        try {
+            CourtCaseEntity courtCase = transcriptionEntity.getCourtCase();
+            transcriptionResponse.setCaseId(courtCase.getId());
+            transcriptionResponse.setCaseNumber(courtCase.getCaseNumber());
+            transcriptionResponse.setCourthouse(courtCase.getCourthouse().getCourthouseName());
+            transcriptionResponse.setDefendants(courtCase.getDefendantStringList());
+            transcriptionResponse.setJudges(courtCase.getJudgeStringList());
+            if (!transcriptionEntity.getTranscriptionDocumentEntities().isEmpty()) {
+                transcriptionResponse.setTranscriptFileName(transcriptionEntity.getTranscriptionDocumentEntities().get(0).getFileName());
+            }
+            transcriptionResponse.setHearingDate(transcriptionEntity.getHearing().getHearingDate());
+            if (transcriptionEntity.getTranscriptionUrgency() != null) {
+                transcriptionResponse.setUrgency(transcriptionEntity.getTranscriptionUrgency().getDescription());
+            }
+            transcriptionResponse.setRequestType(transcriptionEntity.getTranscriptionType().getDescription());
+            transcriptionResponse.setTranscriptionStartTs(transcriptionEntity.getStartTime());
+            transcriptionResponse.setTranscriptionEndTs(transcriptionEntity.getEndTime());
+        } catch (Exception exception) {
+            throw new DartsApiException(TranscriptionApiError.INTERNAL_SERVER_ERROR);
```

Comment:
Please check what @hemantasharma1129  thinks about this one - I reckon we should have a `404` Not Found error normally, but you just need to be careful around the nullable columns in the current schema.

Reply:
Error 500 has now been removed from DMP-1329